gaminatorium-main/adapter/src/test/java/eu/gaminatorium/game/GameControllerTest.java

```diff
@@ -293,12 +293,14 @@ void createGameWhenGameNotExistAndPropertiesAreValid() throws Exception {
             mvc.perform(MockMvcRequestBuilders.post(BASE_URL)
                             .contentType(MediaType.APPLICATION_JSON)
                             .content(newGameJson))
-                    .andExpect(MockMvcResultMatchers.status().isCreated());
+                    .andExpect(MockMvcResultMatchers.status().isOk())   // TODO metoda może zwracać status 201 created
```

Comment:
U can use ststic import (Alt+Enter)
![image](https://github.com/keeeper85/Gaminatorium/assets/93817686/16ade2d6-e7d8-40ae-9c9d-69643f464404)
![image](https://github.com/keeeper85/Gaminatorium/assets/93817686/82479ed6-72ae-4344-8647-62f15e3d9669)

Reply:
and remove unused imports (Alt+Ctrl+o)